modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -264,8 +264,9 @@
                               :inner-join                      true
                               :left-join                       true
                               :nested-fields                   true
-                              :nested-queries                  true
+                              :native-parameter-card-reference false
```

Comment:
Added a new driver feature `:native-parameter-card-reference` to distinguish which the drivers that support card reference template tag parameters (i.e., `:sql` drivers) from ones that support other types of `:native-parameters` (i.e., `:mongo`)

Reply:
I'll add this to the driver changelog once I figure out where this is getting backported to

Reply:
yes it makes it easier to look for specific entries

Reply:
I misread the diff, it's alright.